content/user/first_steps/glossary.md

```diff
@@ -15,6 +15,7 @@ Note: If a term is not in this list, use a web search to learn more about it.  T
 - Document Root - another term for "webroot." 
 - Domain Name - (also "domain") the mnemonic name of your website.  The domain name of this site is `zen-cart.com`.  See [hosting and domain concepts](/user/first_steps/hosting/).  A Name Server converts this mnemonic name into an IP Address. 
 - Downloadable Product - A [downloadable product](/user/products/downloadable/) is a digital item such as an ebook, a music file, an image or a software product.
+- DTK - Short for [Developer's Tool Kit](/user/admin/developers_toolkit/). 
```

Comment:
I can see it making sense in a forum thread where its full name is spelled out at least once beforehand. 
But I recommend we try to avoid ever using "DTK" as an abbreviation in our docs.

Reply:
You used it in the whats new file and I stumbled on it - that's why I added it.